reactivesocket-core/src/main/java/io/reactivesocket/internal/Responder.java

```diff
@@ -75,10 +75,10 @@ private Responder(
         this.isServer = isServer;
         this.connection = connection;
         this.connectionHandler = connectionHandler;
-        this.clientRequestHandler = requestHandler;
+        clientRequestHandler = requestHandler;
```

Comment:
I know that `this.` is not required, but I like the uniformity when we use it all the time.